packages/cf-component-progress/src/index.js

```diff
@@ -1,3 +1,6 @@
-import Progress from './Progress';
+import ProgressUnstyled from './Progress';
+import ProgressTheme from './ProgressTheme';
 
-export default Progress;
+const Progress = applyTheme(ProgressUnstyled, ProgressTheme);
+
+export { Progress as default, ProgressUnstyled, ProgressTheme };
```

Comment:
Yeah I agree, I'm not a fan of the default keyword at all but I figured it would make the migration easier.